ui/src/app/store/machine/types/enum.ts

```diff
@@ -20,6 +20,31 @@ export enum DiskTypes {
   VOLUME_GROUP = "lvm-vg",
 }
 
+export const MachineFormNames = {
```

Comment:
I decided to explicitly list all the different machine forms that can show in the header in order to keep it simpler to start with. Later on we might want to consider extending `NodeActions`.